.eslintrc.json

```
@@ -1,8 +1,13 @@
 {
   "plugins": [
-    "@cypress/dev"
+    "@cypress/dev",
+    "eslint-plugin"
   ],
   "extends": [
     "plugin:@cypress/dev/general"
-  ]
+  ],
+  "rules": {
+    "eslint-plugin/prefer-message-ids": "off",
+    "eslint-plugin/require-meta-docs-url": "error"
```

You'll want to add the rule format so that the correct rule URLs can be enforced and autofixed:
```suggestion
    "eslint-plugin/require-meta-docs-url": ["error", {
      "pattern": "https://github.com/cypress-io/eslint-plugin-cypress/blob/master/docs/rules/{{name}}.md"
    }],
```

---

Some rules don't use documentation from GitHub, so this pattern can't be used.

---

I see. If it's just one rule, then it would probably be good to still use the pattern globally, but just disable the rule for that specific edge case: 

```
url: 'non-github-url' // eslint-disable-line eslint-plugin/require-meta-docs-url
```

---

1/3rd of existing rules don't use GitHub docs. `eslint-disable`s will just clutter up the rules.